### ISSUE TITLE
Not working with ScrollView

### ISSUE THREAD

**Issue by 24dev** [IjzerenHein/react-navigation-shared-element#43]
We have implemented this library exactly as in the documentation, but DetailScreen.js is using a ScrollView as the container. The SharedElement is a child of the ScrollView, like this:

```
<ScrollView style={{ flex: 1 }}>
     <SharedElement id={`${navigation.getParam('id')}`}>
         <View style={{ height: 300, width: 300, backgroundColor: 'grey' }} />
     </SharedElement>
    <View style={{ height: 300, width: 300, backgroundColor: 'grey', marginTop: 30 }} />
</ScrollView>
``` 

The transition occurs perfectly, and the element is shared as expected. However, when scrolling the ScrollView, the 2nd View gets pushed under the SharedElement View. It's like the SharedElement is absolutely positioned at the top of the container, and does not scroll with the rest of the screen. 

Has anyone experienced this before?

**Comment by IjzerenHein** [comment]
Hi. Can you share a video demonstrating the problem?

**Comment by 24dev** [comment]
You can also see that if I begin the navigation.goBack(), then abort it, the ScrollView starts to work as expected. Makes me think I'm doing something wrong in the navigation setup. I've attached my code for that too. 

```

const Stack = createSharedElementStackNavigator(
  {
    List: ListScreen,
    Details: DetailsScreen,
  },
  {
    defaultNavigationOptions: {
      cardStyleInterpolator: ({ current: { progress } }) => {
        return { cardStyle: { opacity: progress } };
      },
    },
  },
);
```

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/17879993/77911109-2fb91980-7288-11ea-9d85-1c8a2bf3c9b6.gif)

**Comment by IjzerenHein** [comment]
I'm not sure I fully understand. Do you mean that the element is stuck after navigating back?
Does this solution help you? https://github.com/IjzerenHein/react-navigation-shared-element/issues/24#issuecomment-599215880

**Comment by 24dev** [comment]
@IjzerenHein  Hmm no doesn't help. My issue is, when in the DetailsScreen, the SharedElement seems fixed to the top of the screen, instead of scrolling with the rest of the screen. You can see the 2nd element on the DetailsScreen scrolls beneath it. I have no styling applied that would make this element fixed position.

The element is fine after navigating back. It's just when you're on the DetailsScreen that it's stuck.

Maybe the video is confusing as both the ListScreen and DetailsScreen look similar. Please watch closely though, it demonstrates the problem. 

Thanks for taking the time to look into this, appreciate it.

**Comment by IjzerenHein** [comment]
When the SharedElement appears stuck, this often means that it cannot find the element (it's waiting for the element to render) on the other screen. Please verify that the `id` that you're using is the same on both screens.

**Comment by 24dev** [comment]
The id is definitely the same, I have hardcoded it to make sure. 

ListScreen
```
<SharedElement id='hardcodedID'>
      <View style={styles.box} />
</SharedElement>


ListScreen.sharedElements = (navigation: any) => {
  return ['hardcodedID'];
};
```

DetailsScreen
```
<SharedElement id='hardcodedID'>
      <View style={styles.box} />
</SharedElement>
```

Did you see how the behaviour goes back to normal when I initiate the goBack() but abort it, so I stay on the same page?  @IjzerenHein

**Comment by 24dev** [comment]
@IjzerenHein Did you have any ideas? I still haven't managed to solve this sadly. If the video is not useful I can try taking a different one.

**Comment by IjzerenHein** [comment]
Hi! No the video isn't making it much clearer for me. Can you share a better video?

**Comment by 24dev** [comment]
Simpler video here: https://github.com/IjzerenHein/react-navigation-shared-element/issues/54

**Comment by IjzerenHein** [comment]
Ah I see. Could you add a test-case for it to the `./example` app?

**Comment by 24dev** [comment]
Solved #54

**Comment by timorss** [comment]
Try set gestureEnabled to false
